refs.bib

```diff
@@ -0,0 +1,8 @@
+@article{Doe2016,
+  author = {Doe, J and Smith, R},
+  journal = {Title of the journal},
+  pages = {10--30},
+  title = {Title of the paper},
```

Comment:
Aha, helpful. That explains my case probably, because I use a custom bibliography style (I rolled my own, though based on elsarticle-harv).